httpclient5/src/test/java/org/apache/hc/client5/http/entity/mime/TestMultipartEntityBuilder.java

```diff
@@ -88,7 +88,7 @@ public void testMultipartCustomContentType() throws Exception {
                 .setLaxMode()
                 .buildEntity();
         Assertions.assertNotNull(entity);
-        Assertions.assertEquals("application/xml; boundary=blah-blah; charset=UTF-8", entity.getContentType());
+        Assertions.assertEquals("application/xml; charset=UTF-8; boundary=blah-blah", entity.getContentType());
```

Comment:
Regardless of your change, I question this test because a multipart request shall start with `multipart/` type...

Reply:
Please clarify what do you mean here. I kept the test to verify the behaviour for the code that did use weird content types for multipart previously.

Reply:
> Please clarify what do you mean here. I kept the test to verify the behaviour for the code that did use weird content types for multipart previously.

This remains as-is. Just a side comment. No change required.